apps/passport-server/src/services/generic-issuance/pipelines/PretixPipeline.ts

```diff
@@ -1963,3 +1963,8 @@ export interface PretixAtom extends PipelineAtom {
   timestampConsumed: Date | null;
   isConsumed: boolean;
 }
+
+export function isPretixAtom(atom: PipelineAtom): atom is PretixAtom {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  return (atom as any).secret !== undefined;
```

Comment:
```suggestion
  return (atom as any).orderCode !== undefined;
```
really a nitpick, but would probably be more robust since we end up checking the orderCode (not the secret)